prow/git/git.go

```diff
@@ -363,3 +361,35 @@ func (r *Repo) MergeCommitsExistBetween(target, head string) (bool, error) {
 	}
 	return len(b) != 0, nil
 }
+
+// ThreeDotDiff does a three dot diff to get all the changes introduced by a
+// commit/branch since it was last synced with the base branch.
+// For more details, see:
+// https://help.github.com/en/articles/about-comparing-branches-in-pull-requests
+func (r *Repo) ThreeDotDiff(base, head string) ([]string, error) {
```

Comment:
Oh wow, today I learned.

We could probably rename the func to `DiffThreeDot` so ppl can find it via their IDE if they type in `Diff`?

Reply:
Thank you for the review. Yes, I think `DiffThreeDot` reads better as well.